Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.jquery.ts

```diff
@@ -569,40 +515,88 @@ function NotNullAssert(v: unknown): asserts v is NotNull {}
                     acceptInput(lastKeyPressCode, focusInputAfterSelection);
                 }
             }
-        }).click(function() {
-
-        }).bind("flushCache", function() {
-            cache.flush();
-        }).bind("collectOptions", function (eventTarget, eventArguments) {
-            const publicOptions: Partial<BocAutoCompleteReferenceValue_CollectedOptions> = {
-              serviceUrl: options.serviceUrl,
-              serviceMethodSearch: options.serviceMethodSearch,
-              serviceMethodSearchExact: options.serviceMethodSearchExact,
-              params : {}
-            };
-            
-            for (const propName in options.extraParams)
-              publicOptions.params![propName] = options.extraParams[propName];
-            
-            Object.assign(eventArguments, publicOptions);
-        }).bind("setOptions", function (eventTarget, eventArguments) {
-            Object.assign(options, eventArguments);
-            // if we've updated the data, repopulate
-            if ("data" in eventArguments)
-              cache.populate();
-        }).bind("collectElements", function (eventTarget, eventArguments) {
-            eventArguments.selectList = select;
-            eventArguments.informationPopUp = informationPopUp;
-        }).bind("showInformationPopUp", function (eventTarget, eventArguments) {
-            informationPopUp.show(eventArguments.message);
-        }).bind("hideInformationPopUp", function() {
-            informationPopUp.hide();
-        }).bind("unautocomplete", function() {
-            informationPopUp.unbind();
-            select.unbind();
-            $(input).unbind();
-            $(input.form!).unbind(".autocomplete");
-        });
+        };
+        input.addEventListener("blur", blurHandler);
+
+        let invalidateResultHandler: Nullable<InvalidateResultEventHandler> = null;
+        let updateResultHandler: Nullable<UpdateResultEventHandler> = null;
+        const publicApi: AutoCompletePublicApi = {
+            invalidateResult: function(handler: InvalidateResultEventHandler): AutoCompletePublicApi {
+                invalidateResultHandler = handler;
+                return this;
+            },
+            updateResult: function(handler: UpdateResultEventHandler): AutoCompletePublicApi {
+                updateResultHandler = handler;
+                return this;
+            },
```

Comment:
Was the jquery API also a single-handler API or could you register multiple handlers?

Reply:
Also single handler